test/built-ins/AsyncGeneratorPrototype/return/return-blocked-broken-promise.js

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2022 Chengzhong Wu. All rights reserved.
```

Comment:
I would suggest renaming this test from "blocked" to "completed", since the other tests are named after the AsyncGeneratorState that they are testing.